tailnet/proto/tailnet.proto

```diff
@@ -0,0 +1,97 @@
+syntax = "proto3";
+option go_package = "github.com/coder/coder/v2/tailnet/proto";
+
+package coder.tailnet.v2;
+
+message UnixTime {
+	// Number of microseconds since Jan 1, 1970 UTC
+	int64 micro = 1;
```

Comment:
I know PostgreSQL only stores microsecond accuracy, but I think it would still make sense for the protocol to maintain the highest level accuracy we can (e.g. `ptypes/timestamp` from https://github.com/golang/protobuf#package-index)

Reply:
You're right, of course.  I'm just being lazy and didn't want to fight with protoc to import stuff from outside the main tree.  I'll sort it out.

Reply:
Awesome! Also, it seems like I linked to something that has been superseded, should've linked to https://pkg.go.dev/google.golang.org/protobuf#section-readme